msteams-platform/apps-in-teams-meetings/teams-together-mode.md

```diff
@@ -9,22 +9,17 @@ ms.topic: conceptual
 > [!NOTE]
 > This feature is currently available in [public developer preview](../resources/dev-preview/developer-preview-intro.md) only.
 
-Custom Together Mode scenes in Microsoft Teams provides an immersive and engaging meeting environment that brings people together and encourages them to turn on their video. It digitally combines participants into a single virtual scene and places their video streams in pre-determined seats designed and fixed by the scene creator.
+Custom Together Mode scenes in Microsoft Teams provide an immersive and engaging meeting environment. It brings people together and encourages them to turn on their video. Custom Together Mode scenes digitally combines participants into a single virtual scene. It places their video streams in pre-determined seats designed and fixed by the scene creator.
```

Comment:
Suggestion: The scenes bring people .. Work on the sentence for subject verb agreement.